.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": [
+    "react-app",
+    "airbnb",
+    "plugin:jsx-a11y/recommended"
+  ],
+  "plugins": [
+    "jsx-a11y",
+    "prettier"
+  ],
+  "rules": {
+    "semi": 0,
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
+  }
+}
```

Comment:
This is missing a new line at the end so the file can breathe..

Reply:
Alright I would add one and send this back up